terra-jupyter-base/Dockerfile

```diff
@@ -121,6 +121,7 @@ RUN pip3 -V \
  && pip3 install python-datauri \
  && pip3 install jupyterlab==0.35.4 \
  && pip3 install cookiecutter \
+ && pip3 install terra-notebook-utils \
```

Comment:
shall we to a version?

Reply:
Yeah, I will. Actual version is pending publishing